### DISCUSSION TITLE
Camera drop down sorting.

### DISCUSSION THREAD

**Discussion by 18923** [blakeblackshear/frigate#12786] [Ideas]
FYI
In the "Review" drop down camera selection,  camera names are in alphabetical order whereas in "Settings" "Camera Settings" the sort order is by "ui: order" as in yaml file.
![sort2](https://github.com/user-attachments/assets/32b5c264-5a1d-494b-8987-ee40deb6546a)
![sort1](https://github.com/user-attachments/assets/c3256203-d8c2-4fb7-849c-580f78c5069a)

**Comment by hawkeye217** [comment]
Thanks, fixed in https://github.com/blakeblackshear/frigate/pull/12789
